README.md

```diff
@@ -1 +1,96 @@
-# Capstone-1
\ No newline at end of file
+#  :open_book: Capstone-1
+This project creates a webpage(home and about page) for a fictional Engineer's Conference. It is based on a design by Cindy Shin in Behance.
+See a short [Project Presentation](https://www.loom.com/share/373c078d61c149fe90f650f1c9799c00)
+
+## :hammer_and_wrench: Build With
+### Tech Stack
+<details><summary>Client</summary>
+<ul>
+  <li><a href="https://html.com/">HTML</a></li>
+  <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+</ul>
+</details>
+<details><summary>Server</summary>
+</details>
+<details><summary>Database</summary>
+</details>
```

Comment:
- Hey, since you've not added a Server and a Database, kindly remove them as they're not relevant to the project to make your readme more professional.